ingestion/src/metadata/ingestion/api/parser.py

```diff
@@ -330,7 +330,17 @@ def parse_workflow_config_gracefully(
         workflow_config = OpenMetadataWorkflowConfig.parse_obj(config_dict)
         return workflow_config
 
-    except ValidationError:
+    except ValidationError as e:
+        error_dict = e.errors()[0]
+        if (
+            error_dict["type"] == "value_error.extra"
+            and error_dict["loc"][0] in OpenMetadataWorkflowConfig.__fields__
+        ):
+            logger.error(
+                f"Failed when parsing the workflow config, extra value {error_dict['loc'][-1]} not permitted : ",
+                e,
```

Comment:
Does it work passing the error as the second parameter?
I have tried this and it throws an exception.
```python
import logging
logger = logging.getLogger("test")
try:
  0/0
except ZeroDivisionError as err:
  logger.error("error ", err)
  # logger.error(f"error: {err}") <-- this works
```